building_a_catalog_simplified.ipynb

```diff
@@ -0,0 +1,8011 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.          "catalog_directory" : "./catalog_demo",
</pre><p>In om3-scripts we just put it into the output folder for a model run. Which maybe is not super useful because we don't want to in  this example </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ACCESS-NRI/access-eval-recipes/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>